python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/name_server.schema.yml

```diff
@@ -8,6 +8,7 @@ type: dict
 keys:
   name_server:
     deprecation:
+      removed: true
       warning: true
       new_key: ip_name_servers
       remove_in_version: v5.0.0
```

Comment:
please remove `keys` and everything below, since it will just take up space in the doc table for something that has been removed.

Reply:
Done